CMSgov/design-system
# [WNMGDS-1535] choice ref update

## Summary

Updating the ChoiceList component to pass through an 'inputRef' defined on a choice.

### Changed

* If an 'inputRef' is defined on a choice in the 'choices' prop, it now gets passed down to the Choice component
* Updating ChoiceList tests to use 'testing-library' instead of Enzyme
* Added unit test to check if an inputRef on a choice i called

## How to test

Run storybook with `yarn storybook`. Navigate to Components > ChoiceList. If you open the dev console, you can see a log that occurs from an inputRef defined on one of the `choices` items
